docs/content/AutoGPT/setup/index.md

```diff
@@ -50,27 +50,40 @@ Since we don't ship AutoGPT as a desktop application, you'll need to download th
 ### Completing the Setup
 
 Once you have cloned or downloaded the project, you can find the AutoGPT Agent in the
-`autogpt/` folder. In this folder:
+`autogpt/` folder.
+Inside this directory you need to configure environment variables in `.env` and optionally `.json` configuration file.
+Environment variables are moslty used for sensitive data like API keys, while `.json` configuration file is used for everything else.
+You can find more information about the environment variables in [Configuration](../configuration/options.md).
```

Comment:
Reword for clarity & typos:
```suggestion
Inside this folder you can configure the AutoGPT application with an `.env` file and (optionally) a JSON configuration file:
* `.env` for environment variables, which are mostly used for sensitive data like API keys
* a JSON configuration file to customize certain features of AutoGPT's components

See the [Configuration](../configuration/options.md) reference for a list of available environment variables.
```

Reply:
I also linked to `components/introduction.md`